### ISSUE TITLE
Mapping C++ constexpr with Java ConstExpr Annotation Library

### ISSUE THREAD

**Issue by eix128** [bytedeco/javacpp#409]
There is new library called constexpr-java.
As i see , currently javacpp doesnt support constexpr.
But you can add support with this library :
https://github.com/junkdog/constexpr-java

**Comment by saudet** [comment]
BTW, JavaCPP does support `constexpr`. It just considers them as normal functions and (immutable) variables, and lets the C++ compiler deal with them. If you encounter bugs related to that though, please let me know.

**Comment by eix128** [comment]
Yeah , but this solution i gave you is faster solution then runtime solution.
Java ConstExpr does all pre calculations for you
